Doc/library/ast.rst

```diff
@@ -140,6 +140,11 @@ Node classes
    In the meantime, instantiating them will return an instance of
    a different class.
 
+.. note::
```

Comment:
I have placed the note on the top, before the classes themselves as we already have a link to the project at the bottom of the page:

> See also Green Tree Snakes, an external documentation resource, has good details on working with Python ASTs.